src/analytics/docs.ts

```diff
@@ -72,10 +72,7 @@ export const eventDocs: Record<AnalyticsEventType, string> = {
   [AppEvents.multichain_beta_opt_out]: `When the user taps the No Thanks button on the multichain beta screen`,
   [AppEvents.multichain_beta_contact_support]: `When the user taps the Contact Support button on the multichain beta screen`,
   [AppEvents.handle_deeplink]: `When a deeplink that leads into the app is detected and handled`,
-  [HomeEvents.hamburger_tapped]: ``,
```

Comment:
Should these be added to the legacy events at the bottom of the file?

Reply:
yeah I thought about it, but the event names are pretty verbose, so ignored them.